endoreg_db/translation.py

```
@@ -0,0 +1,194 @@
+from modeltranslation.translator import TranslationOptions, translator
+from modeltranslation.decorators import register
+
+from endoreg_db.models import (
+    Examination,
+    Finding,
+    FindingLocationClassification,
+    FindingLocationChoice,
+    FindingMorphologyClassification,
+    FindingMorphologyChoice,
+    Intervention
+)
```

Pylint (3.3.7)

[error] No name 'FindingLocationChoice' in module 'endoreg_db.models'

(E0611)